spec/lib/publisher_spec.rb

```diff
@@ -0,0 +1,107 @@
+require "rails_helper"
+require 'gds_api/test_helpers/publishing_api'
+require 'gds_api/test_helpers/rummager'
+
+RSpec.describe Publisher do
+  include GdsApi::TestHelpers::PublishingApi
+  include GdsApi::TestHelpers::Rummager
+
+  before do
+    stub_default_publishing_api_put
+    stub_any_rummager_post
+  end
+
+  context "when publishing a policy" do
+    let(:policy) { FactoryGirl.create(:policy) }
+
+    before do
+      Publisher.new(policy).publish!
+    end
+
+    it "publishes the policy to the Publishing API" do
+      assert_publishing_api_put_item(
+        policy.base_path,
+        ContentItemPresenter.new(policy).exportable_attributes.as_json
+      )
+    end
+
+    it "adds the policy to the rummager search index" do
+      expected_json = JSON.parse({
+        title: policy.name,
+        description: policy.description,
+        link: policy.base_path,
+        indexable_content: "",
```

Comment:
Because we have nothing to put here that wouldn't be in the description: https://github.com/alphagov/policy-publisher/pull/29#discussion_r26200107